client/src/App.css

```diff
@@ -622,11 +622,21 @@ body{
 /* Toast */
 
```

Comment:
Instead of writing all CSS code in one file try to distribute it in separate files for separate components that way it becomes easy to manipulate it at later time

Reply:
yeah done